.github/workflows/dist.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: dist
+
+on: 
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Black
+      run: |
+        python -m pip install black
+        black --check --diff  .
+
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04]
+        python_version: [3.6, 3.7, 3.8]
```

Comment:
Doesn't pynetworktables give a pure py3 wheel?

Reply:
Yes. Should it only build on 3.6 then?

Reply:
I changed it to only build on 3.6.